app/controllers/concerns/current_project.rb

```diff
@@ -0,0 +1,18 @@
+module CurrentProject
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :current_project
+  end
+
+  def current_project
+    @project
+  end
```

Comment:
Functionless methods like this seem wrong to me.
```
def current_project
  @project ||= Project.find_by_param!(params[:project_id])
end
```
And then delete the ```find_project``` below. And delete the before_action find_project everywhere, as it'll only try to find it when a controller tries to access it.